sdk/core/core-client-rest/src/helpers/getBinaryBody.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Converts a string representing binary content into a Uint8Array
+ */
+export function encodeBinaryContent(content: string): Uint8Array {
```

Comment:
when reading this method name in later code I find myself getting confused about what it does (e.g. it could encode binary content as a string) so what if we made the operations a little more explicit?

```suggestion
export function stringToBinaryArray(content: string): Uint8Array {
```